### DISCUSSION TITLE
Log4j CVE-2021-45046

### DISCUSSION THREAD

**Discussion by Magister89** [AbsaOSS/spline#990] [Q&A]
Hi guys,

Is Spline affected by this vulnerability?

Thanks for the support

**Comment by cerveada** [comment]
**No**

## Spline Gateway, Kafka and UI (and their transitive dependencies)
are not using log4j at all.

## Spline Spark Agent
is using logging from Spark. Spark uses older log4j (1.2.17) that is not affected by this.
